### PR TITLE
update account properties,fix issue #676

### PR DESCRIPTION
**Pull request by akurach**
Related Issue: #676 

## New Behavior

Set First_name, Last_name, email, and permissions properties as we set in yml

## Contrast to Current Behavior

First_name, Last_name, email, and permissions properties was not used for account creation

## Discussion: Benefits and Drawbacks

Work as it designed and fixed issue request

## Changes to the Wiki

No

## Proposed Release Note Entry

Fix wrong behaviour while creating users with init startup scripts

## Double Check

* [*] I have read the comments and followed the PR template.
* [*] I have explained my PR according to the information in the comments.
* [*] My PR targets the `develop` branch.

### CODE REVIEW

**Comment by akurach** [comment]
@ryanmerolle  can you review this for starting workflow?

File: startup_scripts/000_users.py [hunk 1]
```diff
@@ -13,6 +13,12 @@
         user = User.objects.create_user(
             username=username,
             password=user_details.get("password", 0) or User.objects.make_random_password(),
+            first_name=user_details.get("first_name", "no-name"),
+            last_name=user_details.get("last_name", "no-lastname"),
+            email=user_details.get("email", "none"),
```
Comment (kkthxbye-code) [inline startup_scripts/000_users.py:18]: The default for `first_name`, `last_name` and `email` should probably be empty string "" as they are allowed to be blank.

File: startup_scripts/000_users.py [hunk 2]
```diff
@@ -13,6 +13,12 @@
...
+            is_staff=user_details.get("is_staff", False),
+            is_active=user_details.get("is_active", False),
```
Comment (kkthxbye-code) [inline startup_scripts/000_users.py]: If `is_active` is to default to False (I don't think it should), then the examples [here](https://github.com/netbox-community/netbox-docker/blob/release/initializers/users.yml) should probably mention that. Not sure preloading a disabled user is common.

File: startup_scripts/000_users.py (hunk 1 above)
Comment (akurach) [reply startup_scripts/000_users.py:18]: fair enough, I'll fix it

**Comment by akurach** [comment]
looks like failed test is broken

**Comment by tobiasge** [comment]
Fixed by PR #729

**Pull request closed without merging**